Utilities/PPS/interface/PPSOpticsCalibrator.h

```diff
@@ -0,0 +1,86 @@
+#ifndef PPS_OPTICS_CALIBRATION
+#define PPS_OPTICS_CALIBRATION
+#include "Utilities/PPS/interface/PPSUtilities.h"
+#include <boost/program_options.hpp>
+#include "TROOT.h"
+#include "Rtypes.h"
+#include "TH2F.h"
+#include "TTree.h"
+#include "TDirectory.h"
+#include "TSystem.h"
+#include "TRandom3.h"
+#include "TFile.h"
```

Comment:
@mundim @civanch it looks like that the comment of @davidlange6 in #24490 has been ignored here, why?